### ISSUE TITLE
0.8.2 does not respect --mf-fps on JPEGs

### ISSUE THREAD

**Issue by Hello71** [mpv-player/mpv#1689]
```
[15:58:26] < Hello71> before 0.8, I used to use mpv --mf-fps=x *.jpg *.png to make a simple slideshow. now, mpv does not honor the fps for jpeg files. how do I set that in 0.8?
[16:00:18] < wm4> Hello71: post a log with -v
[16:00:41] < wm4> Hello71: when playing a jpg file
[16:06:52] < Hello71> wm4: https://bpaste.net/show/1d46339bb1ef
[16:07:15] < wm4> aha
[16:08:36] < wm4> weird
[16:10:47] < wm4> maybe it's specifically broken in the 0.8 releases, but not mpv master
[16:11:43] < wm4> Hello71: can you open an issue in github?
[16:11:55] < wm4> I'll take a look tomorrow
[16:13:25] < Hello71> 0.8.0 works fine, but 0.8.2 doesn't
```

**Comment by wm4** [comment]
The fix will be in 0.8.3. @Kovensky: do you plan to release it soon? You already pushed release notes (I just added my fixup commit after it).

**Comment by Kovensky** [comment]
Just released it.
